app/controllers/teams_controller.rb

```diff
@@ -33,13 +38,22 @@ def create
     authorize @team
     @team.owners << current_user
 
-    if @team.save
-      @team.create_activity(:create,
-                            owner:      current_user,
-                            parameters: { team: @team.name })
-      respond_with(@team)
-    else
-      respond_with @team.errors, status: :unprocessable_entity
+    respond_to do |format|
```

Comment:
@mssola Should I move this to the API too?

Reply:
I was thinking on it but I didn't want to annoy you more with API thingies 😄 I don't have that much spare time in SUSEcon, but I'll try to open up a PR with it asap 👍

Reply:
Same comment as the other. 😃